rerun_py/rerun_sdk/rerun/components/text_log_level_ext.py

```diff
@@ -1,23 +1,35 @@
 from __future__ import annotations
 
-from typing import Final
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from . import TextLogLevel
 
 
 class TextLogLevelExt:
-    CRITICAL: Final = "CRITICAL"
+    CRITICAL: TextLogLevel = None  # type: ignore[assignment]
     """ Designates catastrophic failures. """
 
-    ERROR: Final = "ERROR"
+    ERROR: TextLogLevel = None  # type: ignore[assignment]
     """ Designates very serious errors. """
 
-    WARN: Final = "WARN"
+    WARN: TextLogLevel = None  # type: ignore[assignment]
     """ Designates hazardous situations. """
 
-    INFO: Final = "INFO"
+    INFO: TextLogLevel = None  # type: ignore[assignment]
     """ Designates useful information. """
 
-    DEBUG: Final = "DEBUG"
+    DEBUG: TextLogLevel = None  # type: ignore[assignment]
     """ Designates lower priority information. """
 
-    TRACE: Final = "TRACE"
+    TRACE: TextLogLevel = None  # type: ignore[assignment]
     """ Designates very low priority, often extremely verbose, information. """
+
+    @staticmethod
+    def deferred_patch_class(cls: Any) -> None:
```

Comment:
is it somehow guaranteed that this runs before other classes are initialized? In particular I'm concerned that `LoggingHandler`'s `LVL2NAME` dictionary may still read the none values, not the strings

Reply:
`deferred_patch_class` is called as part of the global import scope, so it should be guaranteed as far as I know yes.